Tools/CISupport/ews-build/steps.py

```diff
@@ -623,14 +627,22 @@ def run(self):
         remote = self.getProperty('github.head.repo.full_name', 'origin').split('/')[0]
         project = self.getProperty('github.head.repo.full_name', self.getProperty('project'))
         branch = self.getProperty('github.head.ref', 'main')
```

Comment:
with so many similar variables, it's better to give them more descriptive names. Some recommendations (feel free to change appropriately):
`branch => target_branch`
`base => base_hash`
`rebase_target => target_rebase_hash or rebase_target_hash`